src/pages/ZNS/ZNS.tsx

```diff
@@ -103,6 +104,7 @@ const ZNS: React.FC<ZNSProps> = ({ domain, version }) => {
 
 	// Force to go back to home if invalid domain
 	React.useEffect(() => {
+		if(loading === false){
```

Comment:
Style nit; This appears to not have been formatted, please make sure to run this through the code styler.